admin/plugins.php

```diff
@@ -0,0 +1,395 @@
+								'action'   => 'perflab_activate_plugin',
+								'plugin'   => $status['file'],
+							),
+							network_admin_url( 'plugins.php' )
+						);
+
+						$action_links[] = sprintf(
+							'<a href="%1$s" class="button activate-now" aria-label="%2$s">%3$s</a>',
+							esc_url( $activate_url ),
+							esc_attr( sprintf( $button_label, $plugin['name'] ) ),
+							$button_text
+						);
+					} else {
+						$action_links[] = sprintf(
+							'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
+							_x( 'Cannot Activate', 'plugin', 'default' )
+						);
+					}
+				} else {
+					$action_links[] = sprintf(
+						'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
+						esc_html( _x( 'Installed', 'plugin', 'default' ) )
+					);
+				}
+				break;
+		}
+	}
+
+	$details_link = esc_url_raw(
+		add_query_arg(
+			array(
+				'tab'       => 'plugin-information',
+				'plugin'    => $plugin['slug'],
+				'TB_iframe' => 'true',
+				'width'     => 600,
+				'height'    => 550,
+			),
+			admin_url( 'plugin-install.php' )
+		)
+	);
+
+	$action_links[] = sprintf(
+		'<a href="%s" class="thickbox open-plugin-details-modal" aria-label="%s" data-title="%s">%s</a>',
+		esc_url( $details_link ),
+		/* translators: %s: Plugin name and version. */
+		esc_attr( sprintf( __( 'More information about %s', 'default' ), $name ) ),
+		esc_attr( $name ),
+		__( 'More Details', 'default' )
+	);
+
+	if ( ! empty( $plugin['icons']['svg'] ) ) {
+		$plugin_icon_url = $plugin['icons']['svg'];
+	} elseif ( ! empty( $plugin['icons']['2x'] ) ) {
+		$plugin_icon_url = $plugin['icons']['2x'];
+	} elseif ( ! empty( $plugin['icons']['1x'] ) ) {
+		$plugin_icon_url = $plugin['icons']['1x'];
+	} else {
+		$plugin_icon_url = $plugin['icons']['default'];
+	}
+
+	/**
+	 * Filters the install action links for a plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string[] $action_links An array of plugin action links.
+	 *                               Defaults are links to Details and Install Now.
+	 * @param array    $plugin       An array of plugin data. See {@see plugins_api()}
+	 *                               for the list of possible values.
+	 */
+	$action_links = apply_filters( 'plugin_install_action_links', $action_links, $plugin );
+
+	$last_updated_timestamp = strtotime( $plugin['last_updated'] );
+	?>
+	<div class="plugin-card plugin-card-<?php echo sanitize_html_class( $plugin['slug'] ); ?>">
+		<?php
+		if ( ! $compatible_php || ! $compatible_wp ) {
+			echo '<div class="notice inline notice-error notice-alt"><p>';
+			if ( ! $compatible_php && ! $compatible_wp ) {
+				esc_html_e( 'This plugin does not work with your versions of WordPress and PHP.', 'default' );
+				if ( current_user_can( 'update_core' ) && current_user_can( 'update_php' ) ) {
+					echo wp_kses_post(
+					/* translators: 1: URL to WordPress Updates screen, 2: URL to Update PHP page. */
```

Comment:
```suggestion
						/* translators: 1: URL to WordPress Updates screen, 2: URL to Update PHP page. */
```

🔢 applies to other places.